tf-modules/ingest/discover-granules-task.tf

```diff
@@ -12,8 +13,10 @@ resource "aws_lambda_function" "discover_granules_task" {
 
   environment {
     variables = {
-      CMR_ENVIRONMENT             = var.cmr_environment
-      stackName                   = var.prefix
+      CMR_ENVIRONMENT               = var.cmr_environment
+      stackName                     = var.prefix
+      system_bucket                 = var.system_bucket
```

Comment:
I'm sure it was added for a reason, but what uses this?

Reply:
You give me too much credit, it's possible this is related to prior implementation work.     Let me track that down.

Reply:
Removed ca03c167b